tutorials/llm/llama-3/pruning-distillation/02_teacher_finetuning.ipynb

```diff
@@ -0,0 +1,153 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "84b146ba-08b6-4adb-a858-8e4294c5e781",
+   "metadata": {},
+   "source": [
+    "\n",
+    "### Step 2: Finetune the teacher on the dataset\n",
+    "\n",
+    "NeMo framework includes a standard python script [megatron_gpt_pretraining.py](https://github.com/NVIDIA/NeMo/blob/main/examples/nlp/language_modeling/megatron_gpt_pretraining.py) for training a model. Once you have your model downloaded and the dataset ready, fine-tuning the teacher model with NeMo is essentially just running this script!\n",
```

Comment:
fix punctuation and capitalization

"NeMo Framework includes a standard Python script, [megatron_gpt_pretraining.py](https://github.com/NVIDIA/NeMo/blob/main/examples/nlp/language_modeling/megatron_gpt_pretraining.py), for training a model. Once you have your model downloaded and the dataset ready, fine-tuning the teacher model with NeMo is essentially just running this script!\n",